experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -99,6 +106,21 @@ def reset(self):
         self.confusion_matrix = np.zeros((self.n_classes, self.n_classes))
 
 
+def normalize(array):
+    """
+    Normalizes a segmentation mask array to be in [0,1] range
+    """
+    min = array.min()
+    return (array - min) / (array.max() - min)
+
+
+def mask_to_disk(mask, fname):
+    """
+    write segmentation mask to disk using a particular colormap
+    """
+    Image.fromarray(cm.gist_earth(normalize(mask), bytes=True)).save(fname)
```

Comment:
Would it be possible to use the same color map that was used to display the training patches in Tensorboard? Not critical, but nice to have. It will make it easier for the user to map the classes in Tensorboard to the ones they see in the output. the function `decode_segmap` already does this (takes class labels and returns a color RGB image suitable for saving/visualization.

Reply:
discussed offline - adding to https://github.com/microsoft/seismic-deeplearning/issues/145